app/src/main/java/com/duckduckgo/app/browser/WebViewVersionProvider.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser
+
+import android.content.Context
+import androidx.webkit.WebViewCompat
+import com.duckduckgo.app.browser.WebViewVersionProvider.Companion.WEBVIEW_UNKNOWN_VERSION
+import com.duckduckgo.di.scopes.AppScope
+import com.squareup.anvil.annotations.ContributesBinding
+import javax.inject.Inject
+
+interface WebViewVersionProvider {
```

Comment:
this is similar to the logic we have in `BrokenSiteActivity`, should we update that class to use this new provider?

Reply:
Hmm, I thought about this but oddly enough the pixel param being sent for broken site reporting expects: `wvVersion=91.0.4472.114` which is passing the whole version not only the major one. Since the use case of this sounds pretty reasonable - as in this is only logged when reporting that a site is broken, is there anywhere we can check to know if we should pass the major version only here as well?